tests/end2end/playwright/attribute-table.spec.js

```diff
@@ -56,19 +58,137 @@ test.describe('Attribute table data restricted to map extent', () => {
+        test(`#${c} Layer export with ${login} user logged in`, {
+            tag: '@write',
+        }, async ({browser}) => {
+            // open admin page to set export permissions
+            const adminContext = await browser.newContext({ storageState: getAuthStorageStatePath('admin') });
+            const page = await adminContext.newPage();
+            const adminPage = new AdminPage(page);
 
-        await page.getByRole('button', { name: 'Export' }).click();
-        await page.getByRole('button', { name: 'GeoJSON' }).click();
+            await page.goto('admin.php');
+            // open maps management page
+            await adminPage.openPage('Maps management');
 
-        getFeatureRequestContains(await getFeatureRequestPromise);
-    });
+            // set layer export permissions
+            await adminPage.modifyRepository('testsrepository');
+            await adminPage.uncheckAllExportPermission();
+            await adminPage.setLayerExportPermission(enabled_groups);
+            await adminPage.page.getByRole('button', { name: 'Save' }).click();
+
+            // login with specific user
+            let userContext;
+            if (login !== '__anonymous') {
+                userContext = await browser.newContext({storageState: getAuthStorageStatePath(login)});
+            } else {
+                userContext = await browser.newContext();
+            }
+            const userPage = await userContext.newPage();
+
+            // go to project page
+            const project = new ProjectPage(userPage, 'enable_export');
+            await project.open();
+
+            // check layer export capabilities for logged in user
+            for(const layerObj of expected){
+                await project.openAttributeTable(layerObj.layer);
+                await expect(userPage.locator('.attribute-layer-action-bar .export-formats')).toHaveCount(layerObj.onPage);
+                await project.closeAttributeTable();
+            }
+
+            // reset layer export permissions
+            await adminPage.modifyRepository('testsrepository');
+            await adminPage.resetLayerExportPermission();
+
+            await adminPage.page.getByRole('button', { name: 'Save' }).click();
+        })
+    })
+
+    test('Layer export request', {
+        tag: '@readonly',
+    }, async ({page}) => {
+        await page.route('**/service/getProjectConfig*', async route => {
+            const response = await route.fetch();
+            const json = await response.json();
+            json.options['limitDataToBbox'] = 'True';
+            await route.fulfill({ response, json });
+        });
+
+        const project = new ProjectPage(page, 'enable_export');
+        await project.open();
+        await project.openAttributeTable('single_wms_points');
+
+        // launche export
+        await project.launchExport('single_wms_points','3.8308870374061232%2C43.5976501168972%2C3.9342777189191174%2C43.647116478434526')
```

Comment:
```suggestion
        // launch export
        const getFeatureRequest = await project.launchExport('single_wms_points','GeoJSON');
                
        const expectedParameters = {
            'SERVICE': 'WFS',
            'REQUEST': 'GetFeature',
            'VERSION': '1.0.0',
            'FORMAT': 'GeoJSON',
            'BBOX': /3.8308\d+,43.597650\d+,3.934277\d+,43.6471\d+/,
        }
        const getFeatureParams = await expectParametersToContain('Export GeoJSON with BBOX', getFeatureRequest.postData() ?? '', expectedParameters)
        //await expect(getFeatureParams.size).toBe(14)
```
Use `expectParametersToContain` to check parameters.